### DISCUSSION TITLE
Homebrew installing not working

### DISCUSSION THREAD

**Discussion by absentio** [streamlink/streamlink#4620] [Q&A] [answered]
Hi, I get error 401 when trying to install Streamlink using Homebrew on Monterey.
I don't know if this error is related to Homebrew or Streamlink

**Comment by bastimeyer** [comment, marked as answer]
Packaging is not done by the Streamlink developers. If you're having any issues with packages, then you'll have to contact the according package maintainers.
https://github.com/Homebrew/homebrew-core/issues

And without any explicit log messages, nobody will be able to help anyway.

Since you're talking about HTTP 401 errors, I had a look at where homebrew is hosting their built packages, and it looks like they are using GitHub's container registry for that:
https://formulae.brew.sh/api/formula/streamlink.json

Accessing the packages does indeed result in a 401 error, so you should contact the homebrew devs about that.

As an alternative to downloading a pre-built package, you can build the package locally from the recipe source in the meantime. See the homebrew docs for that.
https://github.com/Homebrew/homebrew-core/blob/master/Formula/streamlink.rb

**Reply by bastimeyer** [reply]
An issue about that has just been posted:
https://github.com/Homebrew/homebrew-core/issues/104190

**Reply by absentio** [reply]
Thanks a lot @bastimeyer
